index.js

```diff
@@ -324,20 +328,17 @@ module.exports = createStore =>
 				);
 			}
 
-			mounts.set(
-				mountPath,
-				{
-					path,
-					host: host,
-					cache: {
-						ownState: null,
-						viewedState: null,
-						mergedState: null
-					},
-					viewedStateSpec: normalizeViewedStateSpec(viewedStateSpec),
-					reducer: null
-				}
-			);
+			mounts[mountPath] = {
+				path,
+				host: host,
```

Comment:
As long as we're changing this code, I think we can shorten this line to `host,`